patch/kconfig-inclusions

```diff
@@ -35,12 +35,15 @@ CONFIG_SENSORS_MAX31790=m
 CONFIG_EEPROM_OPTOE=m
 ###-> mellanox_amd64-start
 CONFIG_X86_AMD_PLATFORM_DEVICE=y
+CONFIG_MCTP=y
 CONFIG_DMI_SYSFS=y
 CONFIG_OF=y
 CONFIG_EEPROM_AT24=m
 CONFIG_AMD_XGBE=m
 CONFIG_IGB=m
 CONFIG_MLXSW_CORE=m
+CONFIG_USB_NET_DRIVERS=m
+CONFIG_USB_USBNET=m
```

Comment:
Just to confirm, are these two Kconfigs actually needed?

Reply:
Based on https://www.kernelconfig.io/config_usb_usbnet, it seems to be used more for older devices/hardware.

Reply:
updated, removed the kconfig